pkg/store/store.go

```diff
@@ -457,6 +458,12 @@ func (t *Store) ZAdd(zaddOpts schema.ZAddOptions, options ...WriteOption) (index
 	return index, err
 }
 
+func (t *Store) FlushToDisk() {
+	defer t.tree.Unlock()
```

Comment:
Maybe this is not safe. We can have pending writes in worker
try to add https://github.com/codenotary/immudb/blob/master/pkg/store/store.go#L72
Or maybe a better solution can be to use an approach similar to stop method
https://github.com/codenotary/immudb/blob/master/pkg/store/store.go#L65 in which we have 
```
t.wg.Wait()
t.tree.Close()
```